games-emulation/gambatte-jg/gambatte-jg-9999.ebuild

```diff
@@ -15,32 +15,61 @@ if [[ "${PV}" == *9999 ]] ; then
 else
 	SRC_URI="https://gitlab.com/jgemu/${MY_PN}/-/archive/${PV}/${MY_P}.tar.bz2"
 	S="${WORKDIR}/${MY_P}"
-	KEYWORDS="~amd64"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
 fi
 
-LICENSE="BSD GPL-2"
+LICENSE="
+	GPL-2
+	examples? ( 0BSD )
+	jgmodule? ( BSD )
+"
 SLOT="1"
```

Comment:
Just curious, why do you use `SLOT="1"` in your ebuilds?  The default is `SLOT="0"`

Reply:
No, it's ok, I was just curious.  No need to change it :)